### ISSUE TITLE
not working

### ISSUE THREAD

**Issue by YTCloutyyyy** [ZeWhiteHatHacker/new_krunker_hack_sept#8]
:(

**Comment by ZeWhiteHatHacker** [comment]
Yeah it got patched Im working on a new version thats already got ESP.
Subscribe or join the Discord if you want to catch it when its released.
Subscribe: https://bit.ly/BayMaxYT
Discord: https://discord.io/BayMax
